_includes/fe-plugin-intro.md

```diff
@@ -0,0 +1,17 @@
+Frontend plugins provide a way to change the behavior of Deck, Spinnaker's UI
+service. You can add configuration and validation for new stages provided by
+Orca plugins, override existing components with your own implementation, or add
+new Kind definitions for custom resources in your environment.
+
+Plugins are written in any Javascript compatible language (Typescript included),
+and are loaded at runtime through Gate.
+
+The following are examples of extension points that can be overriden:
```

Comment:
I might use a different word than extension point here. Implementing a new stage UI is analogous to implementing a backend extension point. Overriding a component is a little more like overriding a spring bean - it's a little more dangerous and there's a greater chance of danger (e.g. in the case where two plugins are overriding the same component).

Reply:
That may conflict with our definition of `extension point` in the overview doc, or at least the way I interpret the phrasing there:

> An interface defined by one of the Spinnaker services for adding specific functionality.

Should we update that terminology, or update the copy here to choose a different word/phrase?

Reply:
I wasn't clear. I was trying to say that those examples aren't interfaces that you can implement, so they're not really extension points. Those three examples are all what the code calls `OverridableComponents`. Can we call them that? e.g.: "The following are examples of overridable components:"

Reply:
That's my bad! I've rephrased the intro a little more and added example plugins that add new stages as well to demonstrate what's possible.